pyproject.toml

```diff
@@ -94,6 +93,14 @@ exclude = [
 ]
 
 [tool.ruff.per-file-ignores]
-"docs/source/conf.py" = [
+"docs/conf.py" = [
   "A001",
 ]
+"gsw/_fixed_wrapped_ufuncs.py" = [
+  "F403",
+  "F405",
+]
+
+"gsw/_utilities.py" = [
+  "B904",
```

Comment:
For each of these skips, please add a short comment identifying what the rule addresses.

Reply:
Done!